hack/release.sh

```diff
@@ -0,0 +1,46 @@
+#!/bin/bash
+
+# NOTE: this release script is provided for help purpose, but you should carefully read it and understand what it does before running it.
+# Appropriate permissions will be required when pushing git tags / images
+# Adapt the variables below to match your local files, desired version and remote target.
+# Part of this workflow will be automated soon.
+
+path_noo="../network-observability-operator"
+path_gfk="../goflow2-kube-enricher"
+path_plg="../network-observability-console-plugin"
```

Comment:
Maybe we can directly `git clone` these guys if folder doesn't exists and `pull` lasts changes otherwise ?

Who is going to use this script ?

Reply:
yeah we could do that, but my intent was just to providing guidelines for releasing (I hesitate to include this script in the PR or just keep it for myself, but I thought it could be helpful if someone else needs to release before we move to a more automated process)

Reply:
(so I didn't want to invest too much of my time on that :) )

Reply:
sure, I understand. Thanks for the feedback !